src/XFShimmerLayout/Controls/ShimmerLayout.cs

```diff
@@ -354,13 +404,26 @@ private void OnMaskCanvasPaintSurface(object sender, SKPaintSurfaceEventArgs arg
                 paint.Shader = GetGradientShader();
 
                 /* Draw every VisualElement in our layout tree to the SKCanvas */
-                foreach (var view in _childVisualElements)
+                foreach (var skElement in _childVisualElements)
                 {
-                    DrawSKVisualElement(view, args.Surface.Canvas, paint);
+                    //Set cornerRadius and padding for every view
+                    var corner = GetCornerRadiusOverlay(skElement.OriginalView);
+                    var padding = GetPaddingOverlay(skElement.OriginalView);
+
+                    //If not set - set default value
+                    if (corner == 0)
+                        corner = CornerRadiusOverlayDefault;
+
+                    if (padding == 0)
+                        padding = PaddingOverlayDefault;
+
+                    DrawSKVisualElement(skElement, args.Surface.Canvas, paint, corner, padding);
```

Comment:
The PaddingOverlay and the CornerRadius could become properties of SKVisualElement. Rather than passing those as params to DrawSKVisualElement, you could set it to the SKVisualElement